src/Energy.elm

```diff
@@ -6,18 +6,28 @@ module Energy exposing
 {-| An `Energy` value represents an amount of energy (or work) in joules,
 kilowatt hours etc. It is stored as a number of joules.
 
+Note that since `Joules` is defined as `Product Newtons Meters`, you can compute
+force directly as a product of force and distance:
```

Comment:
I think here force should be replaced by energy at the beginning of the line?

Reply:
Yup, good catch, thanks! Fixed now.

I actually think that "make a pull request against my own repository" like I did here might be a good way to solicit feedback from others on proposed changes, without the noise of announcing stuff to *everyone* on Slack or Discourse. Seems like it works =)